STUDIO/org.openl.rules.webstudio/src/org/openl/rules/rest/model/UserCreateModel.java

```diff
@@ -0,0 +1,58 @@
+package org.openl.rules.rest.model;
+
+import java.util.Collection;
+
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+import org.hibernate.validator.constraints.NotBlank;
+import org.openl.rules.rest.validation.UsernameConstraint;
+import org.openl.rules.security.Privilege;
+
+public class UserCreateModel extends UserEditModel {
+
+    @NotBlank(message = "{openl.constraints.user.field.empty.message}")
+    @Size(max = 25, message = "{openl.constraints.user.field.max-length.message}")
+    @Pattern.List({ @Pattern(regexp = "[^.\\s].*[^.\\s]|[^.\\s]", message = "{openl.constraints.username.2.message}"),
+            @Pattern(regexp = "(.(?<![.]{2}))+", message = "{openl.constraints.username.1.message}"),
+            @Pattern(regexp = "[^\\/\\\\:*?\"<>|{}~^]*", message = "{openl.constraints.username.3.message}") })
+    @UsernameConstraint
+    private String username;
+
+    @NotBlank(message = "{openl.constraints.user.field.empty.message}")
+    @Size(max = 25, message = "{openl.constraints.user.field.max-length.message}")
+    private String password;
```

Comment:
Write tests for all beans to check validation

Reply:
Fixed.